core/src/main/scala/org/http4s/StaticFile.scala

```diff
@@ -242,5 +312,5 @@ object StaticFile {
     }
 
   private[http4s] val staticFileKey =
-    Key.newKey[SyncIO, File].unsafeRunSync()
+    Key.newKey[SyncIO, Path].unsafeRunSync()
```

Comment:
This has a slight chance of causing a problem if used in one of the backends outside this repo.  How about a `staticPathKey`?

Reply:
Woah, scary. Isn't the great schism going to be full of bincompat landmines like this?